config/environments/development.rb

```diff
@@ -14,7 +14,7 @@
   config.eager_load = false
 
   # Show full error reports.
-  config.consider_all_requests_local = true
+  config.consider_all_requests_local = false
```

Comment:
We may want to set this back to `true`

Reply:
Whoops good catch. Meant to revert that. 
```suggestion
  config.consider_all_requests_local = true
```